sdk/examples/xo_go/src/sawtooth_xo/handler/handler.go

```diff
@@ -86,6 +86,8 @@ func (self *XoHandler) Apply(request *processor_pb2.TpProcessRequest, context *p
 	switch payload.Action {
 	case "create":
 		return applyCreate(payload.Name, context)
+    case "delete":
```

Comment:
Formatting is off - probably should run `gofmt` on the file.

Reply:
I had run `run_go_fmt fix`, but xo wasn't being checked. I added it.